Started GET "/" for ::1 at 2022-06-29 22:26:11 +0700
  
Mysql2::Error::ConnectionError (Access denied for user 'thao'@'localhost' (using password: NO)):
  
mysql2 (0.5.4) lib/mysql2/client.rb:95:in `connect'
mysql2 (0.5.4) lib/mysql2/client.rb:95:in `initialize'
activerecord (6.0.5) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.5) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.5) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.5) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.5) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.5) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
web-console (4.2.0) lib/web_console/middleware.rb:132:in `call_app'
web-console (4.2.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `catch'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.5) lib/rails/rack/logger.rb:26:in `call'
sprockets-rails (3.4.2) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3.1) lib/rack/method_override.rb:24:in `call'
rack (2.2.3.1) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3.1) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
webpacker (4.3.0) lib/webpacker/dev_server_proxy.rb:23:in `perform_request'
rack-proxy (0.7.2) lib/rack/proxy.rb:67:in `call'
railties (6.0.5) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for ::1 at 2022-06-29 22:26:11 +0700
  
Mysql2::Error::ConnectionError (Access denied for user 'thao'@'localhost' (using password: NO)):
  
mysql2 (0.5.4) lib/mysql2/client.rb:95:in `connect'
mysql2 (0.5.4) lib/mysql2/client.rb:95:in `initialize'
activerecord (6.0.5) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.5) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.5) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.5) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.5) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.5) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
web-console (4.2.0) lib/web_console/middleware.rb:132:in `call_app'
web-console (4.2.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `catch'
web-console (4.2.0) lib/web_console/middleware.rb:17:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.5) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.5) lib/rails/rack/logger.rb:26:in `call'
sprockets-rails (3.4.2) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3.1) lib/rack/method_override.rb:24:in `call'
rack (2.2.3.1) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3.1) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.5) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
webpacker (4.3.0) lib/webpacker/dev_server_proxy.rb:23:in `perform_request'
rack-proxy (0.7.2) lib/rack/proxy.rb:67:in `call'
railties (6.0.5) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.2ms)[0m  [1m[35mCREATE DATABASE `reactapp_development` DEFAULT CHARACTER SET `utf8`[0m
  [1m[35m (0.3ms)[0m  [1m[35mCREATE DATABASE `reactapp_development` DEFAULT CHARACTER SET `utf8`[0m
